addons/medical/ACE_Medical_Treatments.hpp

```diff
@@ -357,6 +385,7 @@ class ACE_Medical_Actions {
             items[] = {};
             treatmentTime = 2.5;
             callbackSuccess = QUOTE(DFUNC(actionRemoveTourniquet));
+            treatmentType = "TourniquetOff";
```

Comment:
These still belong to the UI part don't they?

Reply:
Oh right, forgot those. Will remove them tonight.